runtime/src/accounts_db.rs

```diff
@@ -5653,7 +5653,7 @@ impl AccountsDb {
             let result = self.calculate_accounts_hash_without_index(config, &storages, timings);
 
             // now that calculate_accounts_hash_without_index is complete, we can remove old historical roots
-            self.remove_old_historical_roots(slot, &config.rent_collector.epoch_schedule);
+            self.remove_old_historical_roots(slot, &config.epoch_schedule);
```

Comment:
nit: `config.epoch_schedule` is already a `&EpochSchedule` we don't need the `&`.

Reply:
```suggestion
            self.remove_old_historical_roots(slot, config.epoch_schedule);
```